cloudbeat.yml

```diff
@@ -17,6 +17,14 @@ cloudbeat:
   - name: kube-api
   - name: process
     directory: "/hostfs"
+    processes:
```

Comment:
You shouldn't edit this file directly - It's auto-generated.
Add your changes to the appropriate [template files ](https://github.com/elastic/cloudbeat/tree/main/_meta/config)
Then run `mage config` to generate updated cloudbeat.yml & cloudbeat. reference.yml files

Reply:
fixed

Reply:
looks good to me